patterns/dswp-horizontal-card-large-img-no-shadow.php

```diff
@@ -11,12 +11,12 @@
 <div class="wp-block-group" style="margin-top:var(--wp--preset--spacing--80);margin-bottom:var(--wp--preset--spacing--80);padding-right:var(--wp--preset--spacing--60);padding-left:var(--wp--preset--spacing--60)"><!-- wp:columns {"templateLock":false,"lock":{"move":false,"remove":false},"align":"wide","className":"dswp-horizontal-card-large-img-no-shadow","style":{"border":{"width":"0px","style":"none"}}} -->
     <div class="wp-block-columns alignwide dswp-horizontal-card-large-img-no-shadow" style="border-style:none;border-width:0px"><!-- wp:column {"verticalAlignment":"center","width":"60%","style":{"spacing":{"padding":{"top":"0","bottom":"0","right":"var:preset|spacing|40","left":"0"}}},"layout":{"type":"default"}} -->
         <div class="wp-block-column is-vertically-aligned-center" style="padding-top:0;padding-right:var(--wp--preset--spacing--40);padding-bottom:0;padding-left:0;flex-basis:60%"><!-- wp:heading {"className":"is-style-default","style":{"spacing":{"margin":{"top":"0","bottom":"var:preset|spacing|60"}},"elements":{"link":{"color":{"text":"var:preset|color|font dark"}}}},"textColor":"font dark"} -->
-            <h2 class="wp-block-heading is-style-default has-font-dark-color has-text-color has-link-color" style="margin-top:0;margin-bottom:var(--wp--preset--spacing--60)"><strong>Neque porro quisquamSed</strong></h2>
+            <h2 class="wp-block-heading is-style-default has-font-dark-color has-text-color has-link-color" id="strong-neque-porro-quisquamsed-strong" style="margin-top:0;margin-bottom:var(--wp--preset--spacing--60)"><strong>Neque porro quisquamSed</strong></h2>
```

Comment:
Looks like it's taking the `<strong>` tag from inside the `<h2>`, but then if you remove the strongs from the id it will be the same as the id on line 19.

Reply:
The id's were added by the auto id feature of the plugin.  But whats interesting is you discovered a bug where strong gets added unintentionally.  I have removed the id's as they were not needed in the pattern.